javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,42 @@
+const width = 3;
+const height = 9;
+
+const area = width * height;
+
+console.log('Value of area: ', area);
+console.log('typeof area is ', typeof area);
+
+const bill = 40;
+const payment = 100;
+
+const change = payment - bill;
+console.log('Value of change: ', change);
+console.log('Type of change is', typeof change);
+
+const quizzes = 7;
+const midterm = 9;
+const final = 13;
+
+const grade = (midterm + quizzes + final) / 3;
+console.log('Value of grade: ', grade);
+console.log('Type of grade is ', typeof grade);
+
+const firstName = 'Ethan ';
+const lastName = 'Ostrowski';
+const fullName = firstName + lastName;
+console.log('value of FullName', fullName);
+console.log('Type of fullName ', typeof fullName);
+
+const pH = 4;
+const isAcidic = pH < 7;
+console.log('Value of isAcidic: ', isAcidic);
+console.log('Type of isAcidic is ', typeof isAcidic);
+
+const headcount = 43;
+const isSparta = headcount === 300;
+console.log('Value of isSparta: ', isSparta);
+console.log('Type of isSparta is', typeof isSparta);
+
+const motto = fullName + ' is the GOAT';
```

Comment:
This should be broken up into 2 lines. First `motto` should be initialized with a value of `fullName`, but then using the `+=` operator, you should update it's value with the new string